### ISSUE TITLE
Can't locate object method "route" via package "Mojolicious::Routes::Route"

### ISSUE THREAD

**Issue by eserte** [sshaw/Mojolicious-Plugin-DigestAuth#9]
The test suite started to fail, possibly because of incompatible changes made in Mojolicious 9.x:

```
Can't locate object method "route" via package "Mojolicious::Routes::Route" at t/route.t line 18.
# Looks like your test exited with 255 before it could output anything.
t/route.t .................... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 6/6 subtests 
```

**Comment by aimass** [comment]
same here. Just change ->route to ->_route in line 18 fo route.t
that's it

**Comment by aimass** [comment]
Here is the PR: 

https://github.com/sshaw/Mojolicious-Plugin-DigestAuth/pull/10

And the corresponding commit: https://github.com/sshaw/Mojolicious-Plugin-DigestAuth/pull/10/commits/a03067a8f0cc3d268a6bdaea80c1c6489e08a5f3

**Comment by sshaw** [comment]
PR merged. I will do a release in a day or two.

**Comment by sshaw** [comment]
Released, v0.10: https://metacpan.org/release/SHAW/Mojolicious-Plugin-DigestAuth-0.10
